### PR TITLE
Update images in documentation

### LINKED ISSUE #11904: [Bug]: Documentation info regarding render method of editables is wrong
### Expected behavior

The correct information is displayed

### Actual behavior

The wrong information is displayed

### Steps to reproduce

[The documentation states](https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Editables/index.html):

> Pimcore automatically displays an input widget in the edit mode and renders the content when accessing the document via the frontend.

This is wrong since the returned string value will not be rendered raw. Instead the escaped version of the HTML string is displayed. 
As I see it, the caller is required to add "|raw" after the call to "render" to ensure the correct display in the edit mode.

### PR DESCRIPTION
**Pull request by matthiashamacher**
## Changes in this pull request  
Update images in the documentation with images from Pimcore X (demo.pimcore.fun)

Also Resolves #11904

### CODE REVIEW

**Comment by fashxp** [comment]
just let us know when it is ready for review. thx

**Comment by matthiashamacher** [comment]
Will do. 

Currently there are still at least the following todos:

- [x] 06_Create_a_First_Project.md
- [x] 14_Image.md > Advanced Usage > Thumbnails
- [x] 18_Link.md > Basic Usage
- [x] 40_WYSIWYG.md > Custom configuration for CK Editor
- [x] 50_Localized_Fields.md
- [x] classes-data-inheritance1-3.png
- [x] editables_snippet_add_document.png
- [x] editables_snippet_add_document_popup.png
- [x] editables_snippet_container_empty.png
- [x] editables_snippet_container_filled.png
- [x] editables_snippet_document_type.png
- [x] editables_snippet_editmode.png
- [x] editables_snippet_inc_editmode_preview.png
- [x] editables_table_editmode.png
- [x] editables_video_*.png
- [x] formatter_*.png
- [x] iframe_*.png
- [x] perspectives_custom_perspective.png
- [x] workflow_*.jpg

**Comment by matthiashamacher** [comment]
@fashxp I think I'm done, please have a look. 😀

File: doc/Development_Documentation/04_Assets/03_Working_with_Thumbnails/01_Image_Thumbnails.md [hunk 1]
```diff
@@ -231,21 +231,6 @@ $thumbnail->getHtml([
 Additionally there are some special parameters to [customize generated image HTML code](../../03_Documents/01_Editables/14_Image.md#page_Configuration).
 
 
-## Using ICC Color Profiles for CMYK -> RGB
```
Comment (fashxp) [inline doc/Development_Documentation/04_Assets/03_Working_with_Thumbnails/01_Image_Thumbnails.md:251]: we should not remove that, since the feature is still there - but configuration moved to symfony config. 


```yml

pimcore:

    assets:

        # Absolute path to default ICC RGB profile (if no embedded profile is given)
        icc_rgb_profile:      null

        # Absolute path to default ICC CMYK profile (if no embedded profile is given)
        icc_cmyk_profile:     null

```

File: doc/Development_Documentation/07_Workflow_Management/05_Workflow_Tutorial.md [hunk 2]
```diff
@@ -233,10 +233,12 @@ And, the transition with a *"timeWorked"* field.
         publish:
             from: content_prepared
             to: accepted
-            guard: "is_fully_authenticated() and has_role('ROLE_PIMCORE_SUPERUSER')"
+            guard: "is_fully_authenticated() and is_granted('ROLE_PIMCORE_SUPERUSER')"
```
Comment (fashxp) [inline doc/Development_Documentation/07_Workflow_Management/05_Workflow_Tutorial.md:236]: is that change really necessary?

Comment (fashxp) [reply doc/Development_Documentation/07_Workflow_Management/05_Workflow_Tutorial.md:236]: I think it should be something like `guard: "is_fully_authenticated() and 'ROLE_PIMCORE_SUPERUSER' in role_names"` ?

**Review by fashxp — CHANGES_REQUESTED** [review]

File: doc/Development_Documentation/07_Workflow_Management/05_Workflow_Tutorial.md (hunk 2 above)
Comment (matthiashamacher) [reply doc/Development_Documentation/07_Workflow_Management/05_Workflow_Tutorial.md:236]: `is_granted` is the official replacement of `has_role` therefore I think this is correct in this case: https://symfony.com/blog/symfony-4-2-curated-new-features#security (second bulletpoint)

**Review by fashxp — APPROVED** [review]

**Review by fashxp — APPROVED** [review]

**Comment by fashxp** [comment]
thx very much!
